index.html

```diff
@@ -31,7 +30,6 @@ <h3>Актори:</h3>
                 <li>Том НеХоланд</li>
                 <li>Том і Джері</li>
             </ul>
-            <h3>Рейтинг: 2/3</h3>
         </article>
         <article>
```

Comment:
You can`t have two title in one block, You have situation where this block can be films and actors, not good, try use section for actors